config/config.go

```diff
@@ -336,6 +340,26 @@ func (cfg *Manager) LoadJSONFromFile(path string) error {
 	return err
 }
 
+// LoadJSONFromHTTPSource reads a Configuration file from a URL and parses it.
+func (cfg *Manager) LoadJSONFromHTTPSource(url string) error {
```

Comment:
One thing to keep in mind is that the source `service.json` should not have a `source` element in it.
If it has it will fetch config from there. 
`(LoadJSONFromHTTPSource --> LoadJSON --> LoadJSONFromHTTPSource)` 
And source to next source requests will keep happening until it eventually find a config without `source` in it. 

This also restricts me from places where I can get config from. I can't just use the next node I know which is part of the cluster (actually I can, but it will all lead me to few special or trusted nodes).

Reply:
I have added code to prevent this.

Reply:
and tests